externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/SparkTBinaryFrontendService.scala

```diff
@@ -94,7 +94,13 @@ class SparkTBinaryFrontendService(
   }
 
   override def attributes: Map[String, String] = {
-    Map(KYUUBI_ENGINE_ID -> KyuubiSparkUtil.engineId)
+    Map(
+      KYUUBI_ENGINE_ID -> KyuubiSparkUtil.engineId,
+      // TODO Support Spark Web UI Enabled SSL
+      KYUUBI_ENGINE_URL -> (sc.uiWebUrl match {
```

Comment:
I would like to expose more parameters, likes memory setting.

Reply:
+ 1 for expose more spark info, i will add later.

Reply:
you can add a internal config for the config keys list.

Such as `spark.driver.memory,spark.executor.memory`.

Reply:
and you can only expose the config items that have explicit value.